### PR TITLE
Started Final Report

### PR DESCRIPTION
**Pull request by erichsueh3**

### CODE REVIEW

File: docs/final.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            "flowing_lava": -1
+        }
+```
+Any other block is represented as a 0.
+
+### Actions
+Our agent's actions will consist of discrete movements, including turning left, turning right, moving forward, jumping and moving forward, and attacking. This will be represented by our action dictionary.
+
+### Rewards
+Our agent will be rewarded for mining a variety of materials, including: diamond, gold, iron, emerald, coal, redstone. Higher valued ores will be more scarce within our grid. Lapis will not be represented on our resource list.
```
Comment (lilphunky) [inline docs/final.md:39]:
```suggestion
Our agent will be rewarded for mining a variety of materials, including: diamond, gold, iron, emerald, coal, redstone. Higher valued ores will be more scarce within our grid. It should be noted that mining some ores results in collecting more than 1 of the associated resource. For example, mining 1 redstone ore block results in 5 redstones collected. The rewards below indicate the reward for each individual resource collected.
```

Comment (lilphunky) [reply docs/final.md:39]: I took out the sentence about lapis since we don't generate any lapis blocks anyway

**Review by lilphunky — CHANGES_REQUESTED** [review]
See comment

File: docs/final.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+| Materials | Rewards | Density |
+| ----------- | ----------- | ----------- |
+| Diamond | 6 | 2% |
+| Gold | 5 | 7% |
+| Iron | 4 | 10% |
+| Emerald | 3 | 13% |
+| Coal | 2 | 17% |
+| Redstone | 0.1 | 25% |
+
```
Comment (mickychetta) [inline docs/final.md:50]: add lava density, base it off https://github.com/mickychetta/speed-miners/pull/34

**Review by mickychetta — APPROVED** [review]
good base final report, will edit along the way

**Pull request merged**